### ISSUE TITLE
Request: WordPress plugin

### ISSUE THREAD

**Issue by haneef95** [hybridauth/hybridauth#1219]
## Feature / Question

Does this library have a WordPress plugin? If not could you guys make an official one please?

It would be really helpful.

Thanks

**Comment by ApacheEx** [comment]
yeah, currently, there is no such provider. I hope it's not a big deal to provide one. So, welcome to create PR.

**Comment by champsupertramp** [comment]
Hi @ApacheEx, I have successfully created a WordPress plugin with Hybridauth, but it is developed for a company. I want to develop a WordPress plugin dedicated to Hybridauth. Please let me know how I can contribute. Is a simple PR enough?

**Comment by ApacheEx** [comment]
Hello, yeah, let's start with a simple PR

thanks

**Comment by haneef95** [comment]
That would be amazing @ApacheEx. Kindly state the PR in this thread once it's created.

Thank you
